src/components/editMeasure/populationCriteria/groups/groupPopulations/MeasureGroupPopulationSelect.tsx

```diff
@@ -76,6 +75,9 @@ const MeasureGroupPopulationSelect = ({
           inputProps={{
             "data-testid": `select-measure-group-population-input`,
           }}
+          SelectDisplayProps={{
+            "aria-required": required ? true : false,
```

Comment:
Is the ternary used in case `required` is not a boolean?

Reply:
Yes, this is for that case